Surge no longer compiles with Rack v1-gpl

Following the compile instructions on Kubuntu 18.04.2 system, I get the following new error with "v1-gpl" Rack (v1.dev.09c8527). Previously I was able to successfully compile with "v1" branch of Rack (v1.dev.380d634). But a fresh compile of branch 'v1-gpl' results in errors: 

`make: *** [build/src/SurgeRackGUI.cpp.o] Error 1
make: *** Waiting for unfinished jobs....
../../compile.mk:61: recipe for target 'build/src/SurgeWidgets.cpp.o' failed
make: *** [build/src/SurgeWidgets.cpp.o] Error 1
../../compile.mk:61: recipe for target 'build/src/SurgeVCF.cpp.o' failed
make: *** [build/src/SurgeVCF.cpp.o] Error 1
../../compile.mk:61: recipe for target 'build/src/SurgeWaveShaper.cpp.o' failed
make: *** [build/src/SurgeWaveShaper.cpp.o] Error 1
../../compile.mk:61: recipe for target 'build/src/SurgeVOC.cpp.o' failed
make: *** [build/src/SurgeVOC.cpp.o] Error 1
../../compile.mk:61: recipe for target 'build/src/SurgeWTOSC.cpp.o' failed
make: *** [build/src/SurgeWTOSC.cpp.o] Error 1
../../compile.mk:61: recipe for target 'build/src/SurgeOSC.cpp.o' failed
make: *** [build/src/SurgeOSC.cpp.o] Error 1
In file included from ../../include/common.hpp:2:0,
                 from ../../include/rack.hpp:4,
                 from src/UserInteractionsRack.cpp:2:
src/UserInteractionsRack.cpp: In function ‘void Surge::UserInteractions::promptError(const string&, const string&, SurgeGUIEditor*)’:
../../include/logger.hpp:14:27: error: ‘logger’ has not been declared
 #define WARN(format, ...) logger::log(rack::logger::WARN_LEVEL, __FILE__, __LINE__, format, ##__VA_ARGS__)
                           ^
src/UserInteractionsRack.cpp:18:5: note: in expansion of macro ‘WARN’
     WARN(oss.str().c_str());
     ^~~~
src/UserInteractionsRack.cpp: In function ‘Surge::UserInteractions::MessageResult Surge::UserInteractions::promptOKCancel(const string&, const string&, SurgeGUIEditor*)’:
../../include/logger.hpp:13:27: error: ‘logger’ has not been declared
 #define INFO(format, ...) logger::log(rack::logger::INFO_LEVEL, __FILE__, __LINE__, format, ##__VA_ARGS__)
                           ^
src/UserInteractionsRack.cpp:38:5: note: in expansion of macro ‘INFO’
     INFO(oss.str().c_str());
     ^~~~
../../compile.mk:61: recipe for target 'build/src/UserInteractionsRack.cpp.o' failed
make: *** [build/src/UserInteractionsRack.cpp.o] Error 1
`

--- a/src/Surge.hpp
+++ b/src/Surge.hpp
@@ -8,6 +8,7 @@
 #ifndef RACK_V1
 #define INFO(format, ...)                                                      \
     loggerLog(rack::INFO_LEVEL, __FILE__, __LINE__, format, ##__VA_ARGS__)
+#else
 #endif
 
 extern rack::Plugin *pluginInstance;

--- a/src/SurgeModuleCommon.hpp
+++ b/src/SurgeModuleCommon.hpp
@@ -11,6 +11,10 @@
 #include <map>
 #include <vector>
 
+#if RACK_V1
+namespace logger = rack::logger;
+#endif
+
 struct SurgeModuleCommon : virtual public rack::Module {
 #if RACK_V1
     SurgeModuleCommon() : rack::Module() {  }

--- a/src/SurgeWidgets.cpp
+++ b/src/SurgeWidgets.cpp
@@ -4,6 +4,10 @@
 #include <execinfo.h>
 #endif
 
+#if RACK_V1
+namespace logger = rack::logger;
+#endif
+
 void stackToInfo()
 {
 #if MAC

--- a/src/UserInteractionsRack.cpp
+++ b/src/UserInteractionsRack.cpp
@@ -2,9 +2,7 @@
 #include "rack.hpp"
 #include <sstream>
 
-#ifndef RACK_V1
 using namespace rack;
-#endif
 
 namespace Surge {
 namespace UserInteractions {

